docs/root/configuration/http/http_filters/router_filter.rst

```diff
@@ -360,9 +360,9 @@ HTTP response headers set on downstream responses
 x-envoy-upstream-service-time
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Contains the time in milliseconds spent by the upstream host processing the request. This is useful
-if the client wants to determine service time compared to network latency. This header is set on
-responses.
+Contains the time in milliseconds spent by the upstream host processing the request and the network 
+latency between Envoy and upstream host. This is useful if the client wants to determine service time 
```

Comment:
Superfluous spaces? Probably setting your editor to trim empty spaces will be helpful.